source/AsteroidField.cpp

```diff
@@ -64,17 +64,20 @@ void AsteroidField::Add(const string &name, int count, double energy)
 
 
 
-void AsteroidField::Add(const Minable *minable, int count, double energy, const WeightedList<double> &belts)
+void AsteroidField::Add(const Minable *minable, int count, double energy, double spread,
+	const WeightedList<double> &belts)
 {
 	// Double check that the given asteroid is defined.
 	if(!minable || !minable->GetMask().IsLoaded())
 		return;
 
+	spread = spread > 0. ? spread : 1;
```

Comment:
This prevents you from being able to have 0 spread. Spread defaults to one anyway, so you don't need this.